tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -3643,6 +3770,15 @@ show_techsupport/test_auto_techsupport.py::TestAutoTechSupport::test_sai_sdk_dum
       - "asic_type not in ['mellanox']"
       - "is_multi_asic==True"
 
+#######################################
+#####         smartswitch      #####
+#######################################
+smartswitch:
```

Comment:
Please remove `smartswitch` section, its topology mark is only for `smartswitch`